doc/output-details-and-examples/src/output-files/eplustbl-lt-ext-gt.tex

```diff
@@ -997,6 +997,76 @@ \subsection{Envelope Summary}\label{envelope-summary}
 \bottomrule
 \end{longtable}}
 
+The Exterior Fenestration table has been enhanced to show additional columns of data but an explicit example will not fit well on a page so instead the list of columns is shown below:
```

Comment:
Welllllll, ok.

Reply:
I messed with this for a while, can you think of a better approach?

Reply:
It's fine like it is, but one option might be to try a couple of screen shots of the report (left side and right side) and make them consecutive figures.

Reply:
I will remember next time.